config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -98,6 +98,7 @@ presubmits:
       preset-kind-volume-mounts: "true"
       preset-azure-cred-only: "true"
       preset-azure-anonymous-pull: "true"
+      preset-azure-capz-sa-cred: "true"
```

Comment:
@sonasingh46 I only added the new preset to the optional main PR job for now to minimize impact, once you verify that it works as expected for you we can move it to the other jobs. You can test it by adding `[OPTIONAL]` to the test spec(s) you want to test on your PR.

Reply:
Thank you.